src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -573,4 +579,12 @@ private static CollectionResourceClient createUserManualBlocksStorageClient(
 
     return getCollectionResourceClient(client, context, "/manualblocks");
   }
+
+  private CollectionResourceClient createTemplateNoticeClient(
+    OkapiHttpClient client,
+    WebContext context)
```

Comment:
Please combine these lines together, they do not need to be split into a parameter per line